Purpose: Disable the SUSE flag.
Fixes:   None, made as part of Debian porting.
Status:  Debian specific, showed to upstream.
---
--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@ INSCONF  =	/etc/insserv.conf
 #DESTDIR =	/tmp/root
 #DEBUG	 =	-DDEBUG=1 -Wpacked
 DEBUG	 =
-ISSUSE	 =	-DSUSE
+ISSUSE	 =	-DNOTSUSE
 DESTDIR	 =
 VERSION	 =	1.14.0
 DATE	 =	$(shell date +'%d%b%y' | tr '[:lower:]' '[:upper:]')
